dm/syncer/syncer.go

```diff
@@ -866,7 +866,13 @@ func (s *Syncer) updateReplicationLagMetric() {
 	}
 	if minTS != int64(0) {
 		lag = s.calcReplicationLag(minTS)
+		// this means the syncer is blocked so no job is being processed and the lag is always equal to the last one.
+		// we just increased the lag time by 1 to make this metric looks a bit more normal.
+		if lag == s.secondsBehindMaster.Load() {
+			lag += 1
```

Comment:
Will this cause our delay always be 1(even when there is almost no delay)?

Reply:
yes, do we have some other ways to check if syncer is blocked? 🤔

Reply:
This may not be acceptable, users may kill us 😂

Reply:
MySQL master will send heartbeat event if no event every ticker，can it make some help？

Reply:
ping @GMHDBJD